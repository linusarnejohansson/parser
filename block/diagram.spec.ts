import { Error, IO } from "@cogneco/mend"
import { Source } from "../Source"
import { parse } from "."

describe("block.diagram", () => {
	it("simple", () => {
		const result = parse(new Source(IO.StringReader.create("++\n<svg></svg>++\nFigure Caption."), new Error.ConsoleHandler())) || []
		expect(result.map(node => node.toObject())).toMatchSnapshot()
	})
})
