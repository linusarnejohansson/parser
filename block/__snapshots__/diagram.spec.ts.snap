// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`block.diagram simple 1`] = `
Array [
  Object {
    "class": "Block.Diagram",
    "content": Array [
      Object {
        "class": "Inline.Text",
        "value": "Figure Caption.",
      },
    ],
    "value": "<svg></svg>",
  },
]
`;
